packages-web/signature/.gitignore

```diff
@@ -0,0 +1,17 @@
+/tests/TestProjects/**/.classpath
```

Comment:
Why we need widget specific gitignore file ? We already have root .gitignore which covers some of the cases here. If its not enough maybe move the necessary rules to the root .gitignore

Reply:
We could move some rules to the root, but then we would need to clean up all ignored files in all test projects of the mono repo.